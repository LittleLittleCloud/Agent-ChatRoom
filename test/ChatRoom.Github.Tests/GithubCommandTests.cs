using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ApprovalTests.Namers;
using ApprovalTests.Reporters;
using ApprovalTests;
using Spectre.Console.Testing;
using Xunit;

namespace ChatRoom.Github.Tests;

public class GithubCommandTests
{
    [Fact]
    [UseReporter(typeof(DiffReporter))]
    [UseApprovalSubdirectory("ApprovalTests")]
    public async Task ItShowHelperTextTestAsync()
    {
        var app = new CommandAppTester();
        app.SetDefaultCommand<GithubCommand>(GithubCommand.Description);

        var result = await app.RunAsync("--help");

        Approvals.Verify(result.Output);
    }
}
