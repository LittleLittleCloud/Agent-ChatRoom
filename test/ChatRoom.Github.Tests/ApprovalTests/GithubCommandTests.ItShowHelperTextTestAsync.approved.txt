DESCRIPTION:
Github agents for ChatRoom

The following agents are available:
- issue-helper: A github issue helper agent

USAGE:
    testhost.dll [OPTIONS]

OPTIONS:
    -h, --help               Prints help information
    -v, --version            Prints version information
    -r, --room <ROOM>        The room name to create
    -p, --port <PORT>        The port to listen
    -c, --config <CONFIG>    Configuration file